libraries/botbuilder-ai/src/qnaCardBuilder.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * @module botbuilder-ai
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { Activity, MessageFactory, CardAction, CardFactory } from 'botbuilder-core';
+import { QnAMakerResult } from './';
+
+/**
+ * Provides methods to create activities containing hero cards for showing active learning or multi-turn prompt options for the QnAMakerDialog.
+ */
+export class QnACardBuilder {
+
+    /**
+     * Returns an activity with a hero card attachment, containing buttons for active learning suggestions.
+     * @param suggestionsList List of suggestions to be displayed on hero card.
+     * @param cardTitle Title of the hero card.
+     * @param cardNoMatchText Text for button to be added to card to allow user to select 'no match'.
+     */
+    public GetSuggestionsCard(suggestionsList: string[], cardTitle: string, cardNoMatchText: string): Partial<Activity> {
+        if (!suggestionsList) { throw new Error('Missing suggestionsList'); }
```

Comment:
>!suggestionsList [](start = 12, length = 16)

I'd make this check: !Array.isArray(suggestionsList)

Reply:
Thanks - implemented.